apps/src/code-studio/pd/workshop_dashboard/attendance/workshop_attendance.jsx

```diff
@@ -50,13 +50,20 @@ const WorkshopAttendance = React.createClass({
       loading: true,
       workshopState: undefined,
       sessionAttendances: undefined,
-      adminActions: undefined,
       adminOverride: false,
       saving: false,
       isModified: false
     };
   },
 
+  isAdmin() {
+    return window.dashboard.workshop.permission === "admin";
```

Comment:
This is `admin` status (as compared to something like `UserPermission::admin` or otherwise)? If so, the `permission` is confusing. If not, I'd strongly request that this ability gets renamed, so as not to be confused with `admin` status.